packages/webapp/components/footer/FooterNavBarTabs.tsx

```diff
@@ -82,12 +88,45 @@ const Tab = ({ tab, isActive }: TabProps) => {
   );
 };
 
+const Notifications = ({ active }: { active: boolean }): JSX.Element => {
+  const { unreadCount } = useNotificationContext();
+
+  return (
+    <div className="relative">
+      <BellIcon secondary={active} size={IconSize.Medium} />
+      {!!unreadCount && (
+        <Bubble
+          className={classNames('-right-1.5 -top-1.5 cursor-pointer px-1')}
+        >
+          {getUnreadText(unreadCount)}
+        </Bubble>
+      )}
+    </div>
+  );
+};
+
 export function FooterNavBarTabs({
   activeTab,
 }: FooterNavBarContainerProps): ReactElement {
+  const notificationsNavBar = useFeature(feature.notificationsNavBar);
```

Comment:
Can you check if this component only get's rendered on mobile?
Else we'll need to do enrollment check here as well.

Reply:
FooterNavBarTabs is rendered by FooterNavBar if `showNav` is set to true, which is set to true in FooterNavBarLayout if isMobile.

So, only on mobile.